targets/dummy/dummynode.cpp

```diff
@@ -82,29 +83,33 @@ class AsebaNode: public Dashel::Hub
 		vm.variablesSize = sizeof(variables) / sizeof(sint16);
 	}
 	
-	void listen(int basePort, int deltaPort)
+    Dashel::Stream* listen(int basePort, int deltaPort)
```

Comment:
Indentation problem, it is using space rather than tabs corresponding to 4 spaces each.